src/components/AttendanceView.jsx

```
@@ -217,164 +261,283 @@ function AttendanceView({ events, onBack }) {
+                    let badgeVariant, statusText;
+                  
+                    switch (status) {
+                    case 'yes':
+                      badgeVariant = 'scout-green';
+                      statusText = 'Yes';
+                      break;
+                    case 'invited':
+                      badgeVariant = 'scout-blue';
+                      statusText = 'Invited';
+                      break;
+                    case 'no':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)